hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoTransferHandler.java

```diff
@@ -225,7 +225,9 @@ public void handle(@NonNull final HandleContext context) throws HandleException
         final var transferContext =
                 new TransferContextImpl(context, enforceMonoServiceRestrictionsOnAutoCreationCustomFeePayments);
 
+        // (TEMPORARY) Remove this after diff testing
```

Comment:
Does this comment apply to both L229 and L230 now?  (Also should probably use FUTURE keyword to identify this later.)

Reply:
Both lines can be removed after diff testing.